held/app/src/main/java/com/held/activity/VerificationActivity.java

```diff
@@ -202,15 +211,23 @@ public void failure(RetrofitError error) {
                             UiUtils.showSnackbarToast(findViewById(R.id.root_view), "Some Problem Occurred");
                     }
                 });
-    }
 
-    private void callLoginUserApi() {
-        HeldService.getService().loginUser(mPhoneNo, mPin, new Callback<LoginUserResponse>() {
+    }*/
+
+   /* private void callLoginUserApi() {
+        Log.i("VerificationActivity","In callLoginUserApi()");
+        HeldService.getService().loginUser(mPhoneNo, mPin,"", new Callback<LoginUserResponse>() {
             @Override
```

Comment:
In Verification activity user just verified and profile created so no need to give call to login api